posthog/hogql_queries/web_analytics/stats_table_legacy.py

```diff
@@ -0,0 +1,340 @@
+from posthog.hogql import ast
```

Comment:
this doesn't need reviewing, it's just a copy of the old query file, and will be removed once we're over to the new version